project-koku/koku
# conditionally skip schema unit tests

Fixes [COST-####](https://issues.redhat.com/browse/COST-####)

Changes proposed in this PR:
*
*

Testing Instructions:
1. Run `this command` ...
2. Ingest this data `with this other command` ...
3. Expect to see this result:
   ```
   GET http://localhost:8000/api/endpoint

   {json: response}
   ```
4.
5.

---
Additional Context:
The following show file matching working.
* When just the test schema file has changed:
![image](https://user-images.githubusercontent.com/19751919/152457573-d27d0ce6-d0bc-4427-aba9-e768c75871ef.png)
* when there is a migration: 
![image](https://user-images.githubusercontent.com/19751919/152457659-3285b987-15b4-4b98-a67f-19d00c8d5249.png)
